py_gd/__init__.py

```diff
@@ -19,35 +19,38 @@
     # This only works for Anaconda / miniconda
     # On other  systems, logic needs to be added here.
     libpath = os.path.join(os.path.split(sys.executable)[0], "Library", "bin")
-
+    print(">>> ORG LIBPATH:", libpath)
+    libpath = "D:\\WebGnome\\vcpkg\\installed\\x64-windows\\bin"
+    print(">>> LIBPATH:", libpath)
     # UGLY kludge for Windows: load the libgd dll with ctypes so it can be used by the extension
-    # import ctypes
-    # # note: need to load up the dependencies, too
-    # #       this is a serious kludge!
-    # try:
-    #     libpng = ctypes.cdll.LoadLibrary(os.path.join(libpath,'libpng16.dll'))
-    #     zlib = ctypes.cdll.LoadLibrary(os.path.join(libpath,'zlib.dll'))
-    #     libgd = ctypes.cdll.LoadLibrary(os.path.join(libpath,'libgd.dll'))
-    # except WindowsError as err:
-    #     raise WindowsError("Can't find dlls for libgd, libpng, and libz.\n"
-    #                        "This kludge is only written to support Anaconda installs")
+    import ctypes
+    # note: need to load up the dependencies, too
+    #       this is a serious kludge!
+    try:
+        libpng = ctypes.cdll.LoadLibrary(os.path.join(libpath,'libpng16.dll'))
+        zlib = ctypes.cdll.LoadLibrary(os.path.join(libpath,'zlib1.dll'))
```

Comment:
hmm -- "zlib" vs "zlib1" -- not sure what that's about, but I think conda-forge is shipping "zlib", so that will still need to work.

but these could *maybe* be wrapped in try-except so that it will look for both, and see what it finds.